libraries/AC_Avoidance/AP_OAPathPlanner.h

```diff
@@ -114,6 +117,7 @@ class AP_OAPathPlanner {
     bool _thread_created;           // true once background thread has been created
     AP_OABendyRuler *_oabendyruler; // Bendy Ruler algorithm
     AP_OADijkstra *_oadijkstra;     // Dijkstra's algorithm
+    AP_OART_AStar *_oart_astar;
```

Comment:
we should put a little comment beside this method although it is pretty obvious what it's for.